collections/concepts/what-is-the-proxy.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title:  "What Is the Proxy?"
+categories: concepts
+permalink: /concepts/what-is-the-proxy/
+nav_order: 4
+has_children: true
+has_toc: false
+image:
+    path: https://cdn.basistheory.com/images/seo/guides-opengraph.png
+    width: 1200
+    height: 630
+---
+
+# What Is the Proxy?
+
+It is a common need to share data between software systems via HTTP based APIs, but what if the HTTP request requires a piece of sensitive data that you have tokenized and do not want to access directly within your application?
```

Comment:
This sounds more like the problem it solves rather than the concept/definition of the proxy itself, should we have another title like Why is it needed? or Problem

Reply:
The whole guide is titled `What is the Proxy?` not just this section, and this is starting off with some motivation before diving into the `How It Works` section. I don't want to rename the entire guide - I suppose we could add a sub-section with another name right at the top, but we've typically kicked off the guides with a short intro like this without another sub title.